server/game/GameActions/CaptureAction.js

```diff
@@ -58,16 +58,20 @@ class CaptureAction extends CardAction {
             amount: Math.min(this.amount, player.amber)
         };
         return super.createEvent('onCapture', params, (event) => {
+            let amount = event.amount;
             if (!player.anyEffect('captureFromPool')) {
-                player.modifyAmber(-event.amount);
+                if (amount > player.amber) {
+                    amount = player.amber;
+                }
+                player.modifyAmber(-amount);
             }
 
             let extra = 0;
             if (event.amount > 0) {
```

Comment:
Should be "amount" here `if (amount > 0)`

Reply:
Done!